datafusion/physical-expr-common/src/expressions/binary.rs

```diff
@@ -23,8 +23,8 @@ use std::{any::Any, sync::Arc};
 use crate::expressions::datum::{apply, apply_cmp};
```

Comment:
My hope / dream for the binary operators are that over time we would be able to extract them into functions (e.g so we would have `a + b` --> `add(a, b)`

However, I can't think if any good justification for this yet and I think we have bigger things to focus on than the semantics of operators